site/docs/spark.md

```diff
@@ -35,15 +35,21 @@ Iceberg uses Spark's DataSourceV2 API for data source and catalog implementation
 | SQL insert overwrite                         |           | ✔️                     |                                                |
 
 !!! Note
-    Spark 2.4 can't create Iceberg tables with DDL, instead use the [Iceberg API](../api-quickstart).
+    Apache Spark 2.4 can't create Iceberg tables with DDL, instead use the [Iceberg API](../api-quickstart).
 
 
-## Spark 2.4
+## Apache Spark 2.4
 
-To use Iceberg in Spark 2.4, add the `iceberg-spark-runtime` Jar to Spark's `jars` folder.
+To use Apache Iceberg in Apache Spark 2.4, add the `iceberg-spark-runtime` Jar to Spark's `jars` folder.
 
 Spark 2.4 is limited to reading and writing existing Iceberg tables. Use the [Iceberg API](../api) to create Iceberg tables.
 
+```sh
+./gradlew assemble
+spark-shell --jars runtime/build/libs/iceberg-spark-runtime-fa64197.jar
```

Comment:
I think this should encourage people to use the released version instead of a local build.

Reply:
Ah, I see we have a release :-)